modules/openapi-generator/src/main/resources/python-aiohttp/model.mustache

```diff
@@ -109,6 +109,7 @@ class {{classname}}(Model):
 {{/isMapContainer}}
 {{/isContainer}}
 {{^isContainer}}
+        allowed_values = [{{#isNullable}}None,{{/isNullable}}{{#allowableValues}}{{#values}}{{#isString}}"{{/isString}}{{{this}}}{{#isString}}"{{/isString}}{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]  # noqa: E501
```

Comment:
We are tying to minimize the usage of flake8 E501 and other flake8 opt out lines.
Can you spread this to a multiline statement so we won't need # noqa: E501?